Makefile

```
@@ -0,0 +1,35 @@
+# @todo use this APP_CONTAINER in all commands instead of opendorme_app_1
+APP_CONTAINER=$(docker ps | grep php-fpm | awk '{print $1}')
+DC = docker exec -it opendorme_app_1 /bin/bash -c
+
+connect-app: ## Connect to app container
+	docker exec -it opendorme_app_1 /bin/bash
+
+composer-install: ## Install composer dependencies
+	$(DC) 'composer install'
```

We will need `composer update` much more during development than install.
```suggestion
composer-update: ## Update composer dependencies
	$(DC) 'composer update'
```

---

```bash
composer-update: ## Update composer dependencies
$(DC) 'composer update $(filter-out $@,$(MAKECMDGOALS))'
```

Updating with this so it has more flexiblity like 

`composer update` and `composer update enlight/stash-view` both will work